packages/loader/container-loader/src/container.ts

```diff
@@ -719,8 +719,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     private closeCore(error?: ICriticalContainerError) {
+        function safeAlert(m: string) { try { alert(m); } catch { console.log("FAILED TO ALERT"); } }
```

Comment:
You can do this in ContainerRuntime. That way, we don't need to update the loader layer in the PR deployment.

Reply:
Doh, thanks

Reply:
Or GarbageCollector::dispose might be a better place so all code is in one place.